<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompressImgeListFormClass</class>
 <widget class="GUIFormBase" name="CompressImgeListFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CompressImgeListForm</string>
  </property>
  <widget class="QPushButton" name="pushButtonUpdate">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
   <property name="icon">
    <iconset resource="CompressImageList.qrc">
     <normaloff>:/Qtres/Update.png</normaloff>:/Qtres/Update.png</iconset>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>211</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>211</width>
       <height>340</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>No</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date/Time</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="Scroller"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="CompressImageList.qrc"/>
 </resources>
 <connections/>
</ui>
